controls/V-233520.rb

```diff
@@ -206,25 +194,24 @@
 		  skip 'Requires manual review at this time.'
 		end
 	  else
-		sql = postgres_session(pg_dba, pg_dba_password, pg_host, input('pg_port'))
+		sql = postgres_session(input('pg_dba'), input('pg_dba_password'), input('pg_host'), input('pg_port'))
 	
 		roles_sql = 'SELECT r.rolname FROM pg_catalog.pg_roles r;'
-		roles_query = sql.query(roles_sql, [pg_db])
+		roles_query = sql.query(roles_sql, [input('pg_db')])
 		roles = roles_query.lines
 	
 		roles.each do |role|
-		  next if pg_superusers.include?(role)
+		  next if input('pg_superusers').include?(role)
 	
 		  superuser_sql = 'SELECT r.rolsuper FROM pg_catalog.pg_roles r '\
 			"WHERE r.rolname = '#{role}';"
 	
-		  describe sql.query(superuser_sql, [pg_db]) do
+		  describe sql.query(superuser_sql, [input('pg_db')]) do
 			its('output') { should_not eq 't' }
 		  end
 		end
 	
-		authorized_owners = pg_superusers
-		owners = authorized_owners.join('|')

```

Comment:
I think the `owners` variable needs to be redeclared to address a profile error @damif5

Reply:
Done